brave_init_settings.gni

```diff
@@ -8,10 +8,12 @@ declare_args() {
   # "" for stable channel release.
   brave_channel = ""
 
-  # Update feed url on MacOS
-  # TODO(simonhong): This url is stub. Replace with real update server or your
-  # test server.
-  brave_feed_url = "http://localhost:9090"
-
   brave_dsa_file = "dsa_pub.pem"
 }
+
+_base_update_url = "https://updates-test.bravesoftware.com/sparkle/Brave-Browser"
```

Comment:
@RyanJarv this url should be changed with real url

Reply:
Yeah let's change the domain to updates.bravesoftware.com. (won't resolve now, but should be public in a bit)

Reply:
Went ahead and updated that